src/streaming/controllers/PlaybackController.js

```diff
@@ -248,12 +248,14 @@ function PlaybackController() {
     function getStreamStartTime(ignoreStartOffset) {
         let presentationStartTime;
         let fragData = URIQueryAndFragmentModel(context).getInstance().getURIFragmentData();
-        let fragS = parseInt(fragData.s, 10);
-        let fragT = parseInt(fragData.t, 10);
-        let startTimeOffset = NaN;
+        let startTimeOffset = 0;
```

Comment:
There are a load of `isNaN(startTimeOffset)` checks below here. Why has the initial value changed to `0` - surely, at least if `ignoreStartTimeOffset` is true, it should still be `NaN`?